### PR TITLE
WMS-715 | Mobile view for V2 of Dialog component

### PR DESCRIPTION
**Pull request by segmentationfaulter**
#### :thinking: What?
Adds mobile view for dialog component

#### :man_shrugging: Why?
Since it was missing and [mocks](https://www.figma.com/file/Jra5ja8zLZXxkYTk6gFZsq/Modals?node-id=163%3A53) were not present because Sufyan was on holidays.

#### :pushpin: Jira Issue
[WMS-715](https://paacklogistics.atlassian.net/browse/WMS-715)

### CODE REVIEW

**Comment by PedroHLC** [comment]
:thinking: I think it would be good to add it to percy, you can fake RenderConfig for that, I think Table does it too...

**Comment by segmentationfaulter** [comment]
> thinking I think it would be good to add it to Percy, you can fake RenderConfig for that, I think Table does it too... (And to the showcase)

done

File: showcase/src/Dialog.elm [hunk 1]
```diff
@@ -18,34 +17,79 @@ stories : RenderConfig -> ExplorerUI
...
+dialogMobile =
+    story
+        ( "Dialog Mobile"
+        , Element.column
+            [ Element.height <| Element.px mobileResolution.height
+            , Element.width <| Element.px mobileResolution.width
+            , Element.centerX
+            ]
+            [ iconsSvgSprite
+            , Dialog.dialog demoText.title
```
Comment (PedroHLC) [inline showcase/src/Dialog.elm:58]: Can we use the same "Dialog" (everything until dialogViewV2 in a separated function) between stories?

**Review by PedroHLC — APPROVED** [review]

File: showcase/src/Dialog.elm [hunk 2]
```diff
@@ -18,34 +17,79 @@ stories : RenderConfig -> ExplorerUI
...
+                |> dialogViewV2 mobileCfg
+            ]
+        , { defaultWithMenu | code = code }
+        )
+
+
+demoText : { title : String, body : String, icon : String }
+demoText =
+    { title = "Dialog title"
+    , body = "There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain..."
```
Comment (PedroHLC) [inline showcase/src/Dialog.elm:76]: Never imagined that Lorem Ipsum was this dark...

**Review by kress95 — APPROVED** [review]

**Review by renanpvaz — APPROVED** [review]

File: src/UI/Internal/DialogV2.elm [hunk 3]
```diff
@@ -91,69 +118,32 @@ desktopHeaderRow cfg title icon =
 
 
 mobileView : RenderConfig -> Dialog.Dialog msg -> Element msg
-mobileView cfg (Dialog.Dialog { title, close } { body, buttons }) =
+mobileView cfg (Dialog.Dialog { title, icon } { body, buttons }) =
```
Comment (jouderianjr) [inline src/UI/Internal/DialogV2.elm:121]: I think it's missing the border radius as we have in the desktop version

**Review by jouderianjr — APPROVED** [review]

**Pull request merged**